projects/Mallard/src/authentication/lib/AccessController.ts

```diff
@@ -70,6 +71,15 @@ class AccessController<I extends AuthMap, S extends AuthName<I>> {
         return cachedValidAttempt && Date.now() - cachedValidAttempt < ONE_MONTH
     }
 
+    private logReAuthentication = (connection: string) => {
+        const feature = Feature.SIGN_IN
+        loggingService.log({
+            level: Level.INFO,
+            message: `Previous cached auth expired - re-authenticating user`,
+            optionalFields: { connection, feature },
```

Comment:
We capture a connection with every log. But i think its useful, but maybe we should call it `expectedConnection`?